api/test/integration/import-data/xlsx-uploads/import-data-produce-jobs.spec.ts

```diff
@@ -5,13 +5,18 @@ import { ImportDataModule } from 'modules/import-data/import-data.module';
 import { getQueueToken } from '@nestjs/bull';
 import { importQueueName } from 'modules/import-data/workers/import-data.producer';
 import { ImportDataService } from 'modules/import-data/import-data.service';
+import { TasksRepository } from '../../../../src/modules/tasks/tasks.repository';
```

Comment:
![image](https://user-images.githubusercontent.com/973964/148787632-34091eda-fe39-40bc-9a89-a716bd216d5f.png)

at least when importing from src to test